ffc-pay-alerts/notify/notify-template/get-template-id.js

```diff
@@ -0,0 +1,9 @@
+const templateSchema = require('../../schema/event-schema')
+const { notifyEmailTemplateId } = require('../../config')
+
+const getTemplateId = (message) => {

```

Comment:
Can you add unit tests for this file, i.e. what is returned when the message is:
-  a type with only 1 link
- a type with multiple links
- ''
- {}
- []
- false
- true
- 0
- 1
- undefined
- null

Reply:
Service links are beyond the scope of this ticket. They are only added in anticipation that user may request them being added to email content, happy to remove if complicating issue.